### ISSUE TITLE
fortios_system_global seems broken w/ FortiOS 6.0

### ISSUE THREAD

**Issue by j-lahitte** [fortinet-ansible-dev/ansible-galaxy-fortios-collection#124]
Hi,
My issue is similar to #119. I'm using ansible 4.2.0, collection version 2.1.1.

Whenever I try to run fortios_system_global with *any* parameter on a FortiOS 6.0.11 Fortigate like :
```
-      fortinet.fortios.fortios_system_global:
        system_global:
          admin_server_cert: "{{ fortios_certificate_name }}"
```
or 
```
-      fortinet.fortios.fortios_system_global:
        system_global:
          admin_https_ssl_versions : "tlsv1-2"
```
I get a _TypeError: 'NoneType' object is not iterable_

This doesn't happen with FortiOS 6.4.6.

PR #121 seems to have the right fix, except it has to be duplicated in fortios_system_global.py (and probably any module using flatten_single_path since its code is duplicated inside modules using it).

After applying the proposed fix in fortios_system_global.py, I still get an error when trying to modify the admin-server-cert of the host though, whether using 6.0.11 or 6.4.6 hosts. I only get this behaviour when trying to change the admin-server-cert (ie *not* with admin_https_ssl_versions).

This second error I get is _ansible.module_utils.connection.ConnectionError: Remote end closed connection without response_

And what't really weird to me is that the certificate is actually modified to the one I asked after running the task, even if it gives this error.

Thanks for your help

Full 1st error without fix : 
```
"module_stderr": "Traceback (most recent call last):
  File \"/home/myuser/.ansible/tmp/ansible-local-214229jrmbq1p/ansible-tmp-1626100395.566964-21962-73389945377496/AnsiballZ_fortios_system_global.py\", line 100, in <module>
    _ansiballz_main()
  File \"/home/myuser/.ansible/tmp/ansible-local-214229jrmbq1p/ansible-tmp-1626100395.566964-21962-73389945377496/AnsiballZ_fortios_system_global.py\", line 92, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File \"/home/myuser/.ansible/tmp/ansible-local-214229jrmbq1p/ansible-tmp-1626100395.566964-21962-73389945377496/AnsiballZ_fortios_system_global.py\", line 41, in invoke_module
    run_name='__main__', alter_sys=True)
  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code
    exec(code, run_globals)
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 11046, in <module>
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 11025, in main
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1931, in fortios_system
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1914, in system_global
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1893, in flatten_multilists_attributes
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_fqfvbmgx/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1881, in flatten_single_path
TypeError: 'NoneType' object is not iterable
```

Full second error w/ fix :

```
"module_stderr": "Traceback (most recent call last):
  File \"/home/myuser/.ansible/tmp/ansible-local-20351ew6laodf/ansible-tmp-1626100018.765188-20843-220420694408125/AnsiballZ_fortios_system_global.py\", line 100, in <module>
    _ansiballz_main()
  File \"/home/myuser/.ansible/tmp/ansible-local-20351ew6laodf/ansible-tmp-1626100018.765188-20843-220420694408125/AnsiballZ_fortios_system_global.py\", line 92, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File \"/home/myuser/.ansible/tmp/ansible-local-20351ew6laodf/ansible-tmp-1626100018.765188-20843-220420694408125/AnsiballZ_fortios_system_global.py\", line 41, in invoke_module
    run_name='__main__', alter_sys=True)
  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code
    exec(code, run_globals)
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 11045, in <module>
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 11024, in main
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1930, in fortios_system
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_global.py\", line 1919, in system_global
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible_collections/fortinet/fortios/plugins/module_utils/fortios/fortios.py\", line 386, in set
  File \"/tmp/ansible_fortinet.fortios.fortios_system_global_payload_ab8vlkb8/ansible_fortinet.fortios.fortios_system_global_payload.zip/ansible/module_utils/connection.py\", line 195, in __rpc__
ansible.module_utils.connection.ConnectionError: Remote end closed connection without response
```

**Comment by JieX19** [comment]
Hi @j-lahitte 

We've fixed it in the upcoming release. I will update you once it's released.

Thanks.

**Comment by JieX19** [comment]
Hi @j-lahitte,

The error is fixed in the latest version 2.1.2, please install it using ansible-galaxy collection install fortinet.fortios:2.1.1

Thanks.
